GetEvent/index.ts

```diff
@@ -40,11 +26,19 @@ const getEvent: AzureFunction = async (context: Context, req: HttpRequest, user:
         if (!event) {
             throw new ObjectNotFoundFunctionError(functionName, 'Event', eventId);
         }
+
         const eventNode = await nodeModel.findOne({ id: event.node.id }).exec();
         if (!eventNode) throw new ObjectNotFoundFunctionError(functionName, 'Node', event.node.id);
 
+        const eventClassification = getEventClassification(event);
+        if (!canGetClassifiedEvents(user.role) && eventClassification === Classification.PRIVATE) {
```

Comment:
So it will be more effective
```suggestion
        if (eventClassification === Classification.PRIVATE && !canGetClassifiedEvents(user.role)) {
```